test/test_helper.rb

```
@@ -0,0 +1,117 @@
+  # singleton methods on the mock. Each entry should be method_name => return_value_or_proc.
+  # The arguments passed to those methods will be captured into `capture_hash` under
+  # the method name as a symbol (e.g. capture_hash[:get_and_store_credentials_from_code] = [args...]).
+  #
+  # An optional block is yielded the mock and capture_hash for further customization.
+  #
+  # Usage examples:
+  #   captured = {}
+  #   mock = mock_authorizer_with_credentials(captured, mock_credentials)
+  #
+  #   # with extra method that returns a fixed value
+  #   mock = mock_authorizer_with_credentials(captured, nil, extra_methods: { get_and_store_credentials_from_code: mock_credentials })
```

RuboCop (1.81.1)

[convention] Line is too long. [134/120]

(Layout/LineLength)